client/src/templates/layouts/NavigationLayout.js

```diff
@@ -288,6 +297,20 @@ const NavigationLayout = (props) => {
     { stateKey: "generalFeedback", label: t("feedback.otherFeedback") },
   ];
 
+  useEffect(() => {
```

Comment:
I can't recreate the issue in staging as described or I don't understand it...

But I agree with your point @Zaydme I don't think this is ideal. We shouldn't be fetching the user in this place just for this purpose...

In any case we will be changing navigation significantly with #1767 and then maybe again with some other redesign work so I don't think we should merge this until at least 1767 is done.